src/dealDashboard/dealDiscussion/discussionThread/discussionThread.ts

```diff
@@ -86,7 +89,7 @@ export class DiscussionThread {
     return (discussionsIds.indexOf(this.discussionId) + 1).toString() || "-";
   }
 
-  @computedFrom("isLoading.discussions", "deal.isUserRepresentativeOrLead", "threadComments")
+  @computedFrom("isLoading.discussions", "deal.isUserRepresentativeOrLead", "threadComments", "apiErrorText")
```

Comment:
does apiErrorText really exist?

Can we move the actual text copy into the HTML?

Reply:
argh.. is remnant change.

(next time will also search for variable, instead of just renaming command)